eclass/user-info.eclass

```diff
@@ -37,18 +37,27 @@ egetent() {
 		# lookup by uid/gid
 		local opts
 		if [[ ${key} == [[:digit:]]* ]] ; then
-			[[ ${db} == "user" ]] && opts="-u" || opts="-g"
+			[[ ${db} == "user" ]] && opts=( "-u" ) || opts=( "-g" )
 		fi
 
+		# Handle different ROOT
+		[[ ! -z ${ROOT} ]] && opts+=( -R "${ROOT}" )
```

Comment:
Since ROOT is only valid in `pkg_*` phases, I wonder if we should formally ban the function in other phases.

Reply:
How can I do that ? Just by notify it in the doc string ? or with a check ?

Reply:
Truth is, I'm not sure. Maybe docstring will suffice for now.